demo/examples/cubemap_video.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="ko">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0,user-scalable=no">
+    <title>PanoViewer examples</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans|Zilla+Slab:700" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="../common/css/main.css">
+    <link rel="stylesheet" type="text/css" href="../common/css/example.css">
+    
+    <script src="../release/3.0.0-rc/dist/view360.pkgd.js"></script>
+    <style>
+    #myPanoViewer {
+        position: relative;
+        width: 692px;
+        height: 391px;
+    }
+    .tile {
+        width: 100px;
+    }
+    .tile canvas {
+        width: 100px;
+    }
+    </style>
+</head>
+
+<body>
+    <header>
+        <a href="./">
+            <h1>PanoViewer examples</h1>
+        </a>
+        <h2>360 Video (cubemap)</h2>  
+        <p></p>      
+    </header>
+    <section>
+            <div id="myPanoViewer"></div>
+            <p><a href="https://vimeo.com/244870931">Visions - Burano in VR by Maja Wrońska</a></p>            
+            <video id="pano_video" style="display:none;" controls loop preload="auto" crossorigin="anonymous" autoplay>
+                <source src="./img/cube_3x2.mp4" type="video/mp4" />
+            </video>
+            <div class="tile tile0"></div>
+            <div class="tile tile1"></div>
+            <div class="tile tile2"></div>
+            <div class="tile tile3"></div>
+            <div class="tile tile4"></div>
+            <div class="tile tile5"></div>
+    </section>
+    <script>
+    // create PanoViewer with option
+    var PanoViewer = eg.view360.PanoViewer;
+    var container = document.getElementById("myPanoViewer"); 
+    var videoEl = document.getElementById("pano_video");
+
+    var panoViewer = new PanoViewer(
+        container,
+        {
+            projectionType: "cubemap",
+            video: {
```

Comment:
@happyhj I didn't catch before... I feel it makes confusion... because `video` param tend to have too many types. How about to separate a cubemapConfig

Reply:
thank you for feedback. I will add `cubemapConfig` option as you said.